### DISCUSSION TITLE
Quick tool add/toggle to whitelist

### DISCUSSION THREAD

**Discussion by rosnu** [Cookie-AutoDelete/Cookie-AutoDelete#1556] [Ideas / Feature Requests]
I would appreciate a tool in the toolbar or context menu that allows me to quickly add/toggle the domain of the current page to the whitelist.

**Comment by kennethtran93** [comment]
We already have a toolbar icon to which you can add/remove/toggle the current domain to the lists.
![image](https://github.com/Cookie-AutoDelete/Cookie-AutoDelete/assets/6724477/b4a01c5a-e5f4-40dd-8948-5ce0dd8c979c)

And if you enable the context menu in settings, you can also get the ability to add the current page to a list as well (albeit not as quick).
![image](https://github.com/Cookie-AutoDelete/Cookie-AutoDelete/assets/6724477/1ff993ab-e662-4879-a2af-a06e055ca838)
